examples/tensorflow2/README.md

```diff
@@ -0,0 +1,22 @@
+# Examples
+## Example notebooks
+Please refer to the example notebooks in [Amazon SageMaker Examples repository](https://github.com/awslabs/amazon-sagemaker-examples/tree/master/sagemaker-debugger)
```

Comment:
https://github.com/awslabs/amazon-sagemaker-examples/pull/1102

Reply:
we should have specific link in the doc. It will be easier for user to find it rather than guessing. May be a follow up PR after sagemaker-example is merged. 
tensorflow2/tensorflow2_zero_code_change/tf2-keras-default-container.ipynb  , for now you can point out to this script.

Reply:
Can you create a issue to track this.